src/analyzer.ts

```diff
@@ -203,16 +201,47 @@ export class Analyzer {
               col: scannedDependency.locationOffset.col,
               filename: document.url
             };
-            const scannedDocument = await this._analyzeSource(
-                scannedDependency.type, scannedDependency.contents,
-                document.url, locationOffset,
-                scannedDependency.attachedComment);
-            scannedDependency.scannedDocument = scannedDocument;
-            scannedDependency.scannedDocument.isInline = true;
-            return scannedDocument;
+            try {
+              const scannedDocument = await this._analyzeSource(
+                  scannedDependency.type, scannedDependency.contents,
+                  document.url, locationOffset,
+                  scannedDependency.attachedComment);
+              scannedDependency.scannedDocument = scannedDocument;
```

Comment:
Yeah it's super ugly. I didn't see another point in time where we know the mapping between the inline doc and its scanned document.

Setting whether a scanned document is inline could be done at a number of places though.

Thoughts?